sod-server/src/commands/base/PlaceRoadCommand.ts

```diff
@@ -1,11 +1,12 @@
 import { Command } from '@colyseus/command'
 import { MyRoom } from '../../rooms/MyRoom'
+import { CardNames } from '../../rooms/schema/Card'
 
 interface Payload {
   edgeId: string
   playerId: string
 }
-export class PlaceRoadCommand extends Command<MyRoom, Payload> {
+export class PlaceInitRoadCommand extends Command<MyRoom, Payload> {
```

Comment:
Eina Command til hvønn fíl, og eingin grund til at skriva `Init` ístaðin fyri `Initial` :)

Reply:
Skal Payload interface vera skriva tvær staðni? Skal interface vera onkustaðni meira globalt? ella bara importerast inn frá PlaceRoadCommand?